compiler/optimizer/ValuePropagationCommon.cpp

```diff
@@ -4332,20 +4338,29 @@ void OMR::ValuePropagation::transformConverterCall(TR::TreeTop *callTree)
 
       switch (rm)
          {
-         case TR::sun_nio_cs_ISO_8859_1_Encoder_encodeISOArray:  threshold = 0; break;
-         case TR::java_lang_StringCoding_implEncodeISOArray:  threshold = 0; break;
-         case TR::sun_nio_cs_ISO_8859_1_Decoder_decodeISO8859_1: threshold = 0; break;
-
-         case TR::sun_nio_cs_US_ASCII_Encoder_encodeASCII: threshold = 0; break;
-         case TR::sun_nio_cs_US_ASCII_Decoder_decodeASCII: threshold = 0; break;
-
-         case TR::sun_nio_cs_ext_SBCS_Encoder_encodeSBCS: threshold = 11; break;
-         case TR::sun_nio_cs_ext_SBCS_Decoder_decodeSBCS: threshold = 0; break;
-
-         case TR::sun_nio_cs_UTF_8_Decoder_decodeUTF_8: threshold = 0; break;
-         case TR::sun_nio_cs_UTF_8_Encoder_encodeUTF_8: threshold = 0; break;
-
-         default: threshold = 0; break;
+         case TR::sun_nio_cs_ISO_8859_1_Encoder_encodeISOArray:
+         case TR::java_lang_StringCoding_implEncodeISOArray:
+         case TR::sun_nio_cs_ISO_8859_1_Decoder_decodeISO8859_1:
+            threshold = 0;
+            break;
+         case TR::sun_nio_cs_US_ASCII_Encoder_encodeASCII:
+         case TR::sun_nio_cs_US_ASCII_Decoder_decodeASCII:
+         case TR::java_lang_StringCoding_implEncodeAsciiArray:
+            threshold = 0;
```

Comment:
Any particular reason why we have this in group of 3? Just a question (given the previous state this looks better), but I suggested to bring all methods that has threshold 0 together.

Reply:
I misread your suggestion and thought you wanted it to be separated like this as well. It does keep similar named/functioning methods together, but you're right in that it's still got some redundancy. I can change it again if you prefer your original suggestion. Let me know. Thanks!

Reply:
I've grouped them all together with the default case and added a comment to explain why the names are still present in the codebase: https://github.com/eclipse/omr/commit/81f30e4f8eaafb7905c39ee02214ded71d3be315